fs/void-musl/info

```diff
@@ -0,0 +1,17 @@
+#!/bin/false
+
+# basic info
+FS_FANCYNAME="Void-musl"
+#FS_HAS_OOBE="true"
+FS_INST_PKG="/bin/xbps-install -y"
+FS_UPDATE="/bin/xbps-install -Syu"
+
+# bootloader packages
+FS_PKGS_CD_BOOTFW_UBOOT="u-boot-tools curl wget"
```

Comment:
uhh, maybe we should make `FS_PKGS_CD_BASE`

Reply:
needs to be added to debian as well